### ISSUE TITLE
PHP Fatal error:  Uncaught Error: Call to undefined method Sentry\State\Hub::getCurrent() 

### ISSUE THREAD

**Issue by mpjraaij** [getsentry/sentry-php#1208]
I'm trying to create my own transport per https://docs.sentry.io/platforms/php/configuration/transports/ however am getting the error of call to undefined method getCurrent();

```php
$transportFactory = new \Intersolve\Sentry\Transport();

$builder = \Sentry\ClientBuilder::create([
    'dsn' => $_ENV['SENTRY_DSN'],
    'environment' => $_ENV['SENTRY_ENV']
]);
$builder->setTransportFactory($transportFactory);

\Sentry\State\Hub::getCurrent()->bindClient(
    $builder->getClient()
);
```

What am I doing wrong?

**Comment by stayallive** [comment]
Hi there,

I believe the docs are wrong here, let us figure it out so we can update them!

I believe it should be this:

```php
$transportFactory = new \Intersolve\Sentry\Transport();

$builder = \Sentry\ClientBuilder::create([
    'dsn' => $_ENV['SENTRY_DSN'],
    'environment' => $_ENV['SENTRY_ENV']
]);
$builder->setTransportFactory($transportFactory);

\Sentry\SentrySdk::getCurrent()->bindClient(
    $builder->getClient()
);
```

**Comment by mpjraaij** [comment]
@stayallive Thank you for your code. That definitely helped.

Just to be complete it has to be `getCurrentHub`:

```php
\Sentry\SentrySdk::getCurrentHub()->bindClient(
    $builder->getClient()
);
```

**Comment by stayallive** [comment]
Excellent, I'll get on a PR for the docs to change that right away! Will update this issue once that is done, thanks!

**Comment by stayallive** [comment]
@LorenzoRogai wow... that nice comment looking code 😄 (it took me longer than I care to admit to find what you were referencing)

Should be fixed with: getsentry/sentry-docs#3553, thanks 👍
